src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
 body {
+  background-color: #fff;
+  display: flex;
+  justify-content: center;
```

Comment:
Those are not reseting styles

Reply:
that really wasn't my intention, just generic parameters for file